### ISSUE TITLE
About Criterium "Continuous Integration"

### ISSUE THREAD

**Issue by jgroenen** [standard-for-public-code/standard-for-public-code#1053]
In several places 'continuous integration' is mentioned as a criterium and good practice. When translating to Dutch, this term is difficult to translate ('continue integratie' or 'doorlopende integratie'). Since we were replacing more jargon terms (like 'commit') to make it less 'githubby', it made me think: the rest of the Standard is quite agnostic about specific technology and ways of working (it doesn't mention Agile for example), would it perhaps be better to not refer to 'continuous integration' as a term, but as a practice: keep an eye on the changes other people are making and integrate them regularly into your own work?

**Comment by jgroenen** [comment]
In Dutch I replaced the title with "Test for conflicts regularly" ("Test regelmatig op conflicten").

**Comment by ericherman** [comment]
I do not see "Continuous Integration" as being "githubby", as it has been in wide use since at least year 2000 with the rising popularity of agile development; wikipedia places it a decade prior: https://en.wikipedia.org/wiki/Continuous_integration#History

It *is* a domain specific term -- agile development jargon indeed -- for the practice of frequent (successful) merging of changes in to the shared development branch of the codebase. 

Thinking on this for a few minutes, I was not able to come up with a less "jargony" and also more clear term for describing this wide-spread practice.

**Comment by ericherman** [comment]
At the time of this writing, there happens to be a German wikipedia.org page for ( https://de.wikipedia.org/wiki/Kontinuierliche_Integration ), but I do not find a Dutch page. Is this perhaps because the English term is used directly as borrowed words when speaking in Dutch?

**Comment by jgroenen** [comment]
Yes, we had some discussion on using 'continue integratie' or 'doorlopende integratie'. English term is used most of the time, usually by developers (same as with commit and pull request).

**Comment by Ainali** [comment]
On a related note, I wouldn't recommend translating certain terms (like commit and pull request) for the Swedish translation, as those words are used in everyday language and "force inventing" a term would sound funny.

**Comment by jgroenen** [comment]
We discussed this again a bit just now, so I'd like to bump this discussion. In Dutch we decided upon "Test for conflicts regularly", which is nice, because it applies a bit better (in Dutch at least) to all code (including both policy-policy and policy-source code).
